recipes/cuda-cudart/meta.yaml

```diff
@@ -0,0 +1,106 @@
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cudart/{{ platform }}/cuda_cudart-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c0e16e9593a69afe4997ae96a8b024e751ebce0254523fd969b08bb028582d23  # [linux64]
+  sha256: d785fd613e35e3761fdb0016a31a56417f570605cbfb9a93ca9dc68be6789188  # [ppc64le]
+  sha256: 5de248c6c203f6c91ce34256bbfc028bc87c1b0c552f9f51edb65e60ad665f63  # [aarch64]
+  sha256: 1cd8516feee068c6d718453e76b0dfcbee27844a35d4f3aa608a3d316190309c  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+  run_exports:                # [linux]
+    strong:                   # [linux]
+      - __glibc >=2.17        # [linux]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+  run:
+    - sysroot_{{ target_platform }} 2.17   # [linux]
+    - __glibc >=2.17                       # [linux]
+
+outputs:
+  - name: cuda-cudart
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcudart.so.{{ version }}     # [linux]
+        - if not exist %LIBRARY_BIN%\cudart64_*.dll exit 1   # [win]
+    files:
+      - lib/*.so.*                # [linux]
+      - targets/*/lib/*.so.*      # [linux]
+      - Library\bin               # [win]
+
+  - name: cuda-cudart-dev
+    run_exports:
+      weak:
+        - {{ pin_subpackage("cuda-cudart", max_pin='x') }}
+    requirements:
+      run:
+        - cuda-cudart >={{ version }}
+        - cuda-cudart-static >={{ version }}
+    test:
+      commands:
+        - test -d $PREFIX/lib/pkgconfig                  # [linux]
+        - if not exist %LIBRARY_LIB%\cudart.lib exit 1   # [win]
+    files:
+      - lib/*.so                                # [linux]
+      - targets/*/lib/*.so                      # [linux]
+      - include                                 # [linux]
+      - targets/*/include/*                     # [linux]
+      - targets/*/include/cooperative_groups/*  # [linux]
+      - lib/pkgconfig                           # [linux]
+      - Library\include                         # [win]
+      - Library\lib\cudadevrt.lib               # [win]
+      - Library\lib\cuda.lib                    # [win]
+      - Library\lib\cudart.lib                  # [win]
+
+  - name: cuda-cudart-static
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcudart_static.a             # [linux]
+        - if not exist %LIBRARY_LIB%\cudart_static.lib exit 1   # [win]
+    files:
+      - lib/*.a                        # [linux]
+      - targets/*/lib/*.a              # [linux]
+      - Library\lib\cudart_static.lib  # [win]
+
+  - name: cuda-driver-dev
+    build:
+      skip: True                        # [not linux]
+    test:
+      commands:
+        - test -f $PREFIX/lib/stubs/libcuda.so
+    files:
+      - lib/stubs/libcuda.so
+      - targets/*/lib/stubs/libcuda.so
```

Comment:
@robertmaynard, does this help clarify how `libcuda.so` is handled? Anything else we should think about here?

Reply:
yes it does.